### ISSUE TITLE
Случались ли ложноположительные срабатывания антивирусов?

### ISSUE THREAD

**Issue by vtereshkov** [AntKrotov/oberon-07-compiler#12]
Сожалею, что пришлось открыть Issue, но не знаю других способов связаться. 
Сталкивались ли вы с ложноположительным срабатыванием антивирусов на программах, откомпилированных вашим компилятором? Я разрабатываю свой любительский компилятор (https://github.com/vtereshkov/xdpw), и эта проблема с антивирусом AVG мне иногда досаждает.

**Comment by AntKrotov** [comment]
Да, есть такая проблема, не только у меня, но и у других любительских компиляторов (в основном на Windows). Реакция антивируса может непредсказуемо меняться при небольшом изменении в компиляторе. Бороться с этим бесполезно, к сожалению (как я считаю). Разве что, писать разработчикам антивирусов.

**Comment by caufman** [comment]
Есть некоторые сообщества любителей разработки компиляторов и среди них эта проблема известна. Можете спросить в сообществе среды разработки Компонентного Паскаля BlackBox - была точно такая же проблема у Ивана Денисова и он ее как-то победил. [Рокет-чат](https://chat.obertone.ru/home), [Телеграмм-чат](https://t.me/oberonf), [еще один](https://t.me/bbcb_support)
Ну или [вот в этом телеграмм-чате](https://t.me/CompilerDev) еще можете узнать. Здесь более 400 человек примерно с теми же проблемами.
